build.rs

```diff
@@ -231,9 +231,17 @@ fn windows_support() {
 #[cfg(not(target_os = "windows"))]
 fn windows_support() {}
 
+fn ruby_version_features(version: &str) {
+    if version >= "2.4.0" {
```

Comment:
How is this string comparison working with semver?  Is it comparing byte values from left to right?

Reply:
It's not ideal, but yes, it just compares byte values from left to right.
It should be possible to add semver crate and check versions properly.